services/workflows-service/src/workflow/workflow-collection-flow.controller.ts

```
@@ -0,0 +1,82 @@
+  @common.Put('/:workflowId/state')
+  async updateCollectionFlowState(
+    @CurrentProject() projectId: TProjectId,
+    @common.Param('workflowId') workflowId: string,
+    @common.Body() body: UpdateCollectionFlowStateDto,
+  ) {
+    try {
+      return await this.collectionFlowStateService.updateCollectionFlowState(workflowId, body, [
+        projectId,
+      ]);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)